### ISSUE TITLE
Configuring synced folders opens biometric auth several times, crashes when cancelled

### ISSUE THREAD

**Issue by hurricanerainbow** [nextcloud/android#10791]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

(NOT in order)

- Click notification 'configure'
- Have biometric locking enabled
- Have folders show up from another app, that nextcloud app is tracking
- Notification comes up because of it

No, i did not check if there was a similar bug. Didn't want to lose the clipboard contents. Posting from mobile.

### Expected behaviour

- Click configure
- Verify once via biometrics, not once per file (is that what's happening? Why? I don't know the code)
- Configure settings at ease, maybe have a small sandbox session to maintain security (is that too much for an app like this to handle? Good question, i dunno)

### Actual behaviour

I covered it earlier, AFAICT (as far as i can tell)

Oh, something new. It's behaving as expected if you open from the app icon. It didn't when i clicked the notification.

### Android version

13

### Device brand and model

pixel 4 xl

### Stock or custom OS?

Stock

### Nextcloud android app version

20221004

### Nextcloud server version

latest stable release

### Using a reverse proxy?

No

### Android logs

`### Cause of error
```java
Exception in thread "main" java.lang.IllegalStateException: Can not perform this action after onSaveInstanceState
    at androidx.fragment.app.FragmentManager.checkStateLoss(FragmentManager.java:1610)
    at androidx.fragment.app.FragmentManager.enqueueAction(FragmentManager.java:1650)
    at androidx.fragment.app.BackStackRecord.commitInternal(BackStackRecord.java:341)
    at androidx.fragment.app.BackStackRecord.commit(BackStackRecord.java:306)
    at androidx.fragment.app.DialogFragment.show(DialogFragment.java:279)
    at com.owncloud.android.ui.activity.SyncedFoldersActivity.onSyncFolderSettingsClick(SyncedFoldersActivity.kt:584)
    at com.owncloud.android.ui.activity.SyncedFoldersActivity$load$1$1.invokeSuspend(SyncedFoldersActivity.kt:327)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
    at android.os.Handler.handleCallback(Handler.java:942)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:201)
    at android.os.Looper.loop(Looper.java:288)
    at android.app.ActivityThread.main(ActivityThread.java:7898)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)

```

### App information
* ID: `com.nextcloud.android.beta`
* Version: `20221004`
* Build flavor: `versionDev`

### Device information
* Brand: `google`
* Device: `coral`
* Model: `Pixel 4 XL`
* Id: `TP1A.221005.002`
* Product: `coral`

### Firmware
* SDK: `33`
* Release: `13`
* Incremental: `9012097`
`

Pasted from clipboard report button

o/ o7

Have an awesome day, love the work you folks do 💜💚

**Comment by AlvaroBrey** [comment]
@ericcotelnu the crash trace happened after you canceled the biometric auth?

**Comment by joshtrichards** [comment]
Duplicate of #12970 
Fixed in #13037
